test/funnels/Swapper.t.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
```suggestion
// SPDX-License-Identifier: AGPL-3.0-or-later
```

Reply:
Implemented in https://github.com/makerdao/dss-allocator/pull/15